cmd/root.go

```diff
@@ -125,6 +130,21 @@ func initConfig() {
 
 	viper.WatchConfig()
 
+	if slices.Contains(os.Args[1:], "wallet") &&
+		(slices.Contains(os.Args[1:], "new") || slices.Contains(os.Args[1:], "migrate")) {
+		// Skip migration check
+	} else {
+		err = checkWalletMigrated()
+		if err != nil {
+			logFatal(err)
+		}
+
+		err = checkUnencryptedPrivateKeys()
+		if err != nil {
+			log.Println(err)
+		}
+	}
+
```

Comment:
Can we refactor this to use native `cobra` cmds? We're just checking `glif wallet new` and `glif wallet migrate` here right? - any reason to deviate?

Reply:
It's a chicken-and-egg thing ... this code gets executed before cobra is done initializing, so I had to use the raw command line args instead.